### PR TITLE
Add topic: MariaDB

### PR DESCRIPTION
**Pull request by robinnewhouse**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created either the topic I am modifying/adding or the collection entry I am modifying/adding.

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

A curated MariaDB topic is useful considering its increasing adoption across major companies and projects.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by dnk83** [comment]
Super

File: topics/mariadb/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+display_name: MariaDB
+github_url: https://github.com/MariaDB/server
+logo: mariadb.png
+released: October 29, 2009
+short_description: MariaDB is a community-developed, open source relational database management system forked from MySQL.
+topic: mariadb
+url: https://mariadb.org/
+wikipedia_url: https://en.wikipedia.org/wiki/MariaDB
+---
+MariaDB is an open source relational database management system forked from the free MySQL database in 2009 by original MySQL developers. While maintaining high compatibility with MySQL, MariaDB adds new storage engines and features as a community-driven open source alternative. It has gained widespread adoption, being utilized by major companies and many Linux distributions.
```
Comment (jmeridth) [inline topics/mariadb/index.md:13]: Nit: Newline at end of file please

**Review by jmeridth — COMMENTED** [review]
LGTM. Thank you for your contribution. Only one nit. 😄

File: topics/mariadb/index.md (hunk 1 above)
Comment (robinnewhouse) [reply topics/mariadb/index.md]: 😭 newbie mistake. Fixed (as well as a trailing whitespace 😬)

**Review by jmeridth — APPROVED** [review]
LGTM. Thank you for the contribution.

**Pull request merged**